ODIN_II/SRC/odin_util.cpp

```diff
@@ -909,6 +909,49 @@ void trim_string(char* string, const char* chars) {
     }
 }
 
+/*
+ * Will seperate the headers from the inputs and compare them
+ * The trim function is used to elimnate following spaces which
+ * caused a bug. Will return value of 1 if they match and 0 if
+ * they don't match
+ */
+bool output_vector_headers_comp(char * buffer1, char * buffer2)
+{
+    char *ptr1 = (char*)malloc((strlen(buffer1)+1)*sizeof(char));
+    char *ptr2 = (char*)malloc((strlen(buffer2)+1)*sizeof(char));
+
+    strcpy(ptr1,buffer1);
+    strcpy(ptr2,buffer2);
+
+    char * temp1 = NULL;
+    char * temp2 = NULL;
+
+    temp1 = strtok(ptr1,"\n");
```

Comment:
I dont think that is what you want to do.
this will give you one big token being the whole line here.
```
a b c
```

would not match
```
a              b           c
```

whit what you have right now

Reply:
You're right I'll fixe that, also how do you know what order the outputs will be produced?